### ISSUE TITLE
Instructions on how to use.

### ISSUE THREAD

**Issue by bsutton** [letmaik/vscode-git-tree-compare#68]
I've just installed this extension and I can't work out how to use it.

I can't find any instructions (except for the settings).

There is a screen shot that seems to show a window labelled 'GIT TREE COMPARE' but that is not showing up anywhere in my vs code ui.

So I'm not certain whether I don't know how to use the plugin or whether its simply not working.

Some instructions would help either way :)

**Comment by letmaik** [comment]
Right, it may make sense to update the screenshot to show more context on where that actually is. It's in the source control tab and should appear at the bottom somewhere if you have a git repo open. Let me know if that helps.

**Comment by bsutton** [comment]
I suspect it wasn't showing up yesterday from some reason.
Today its visible, but yes an image that shows it in context or at least an explanation.

Your intro also states:
> In bigger projects with many files it also provides context

The word 'context' is bolded which makes me think there is something else that the package does that isn't full explained. The following text talks about comparing branches but doesn't explain how to do it.

Some more details and maybe a few more graphics would make the package much easier to start using.

Brett.

**Comment by HughGrovesArup** [comment]
Just knowing it was on the source control tab was enough for me to see it and figure it out - was at a loss before finding this thread!

**Comment by keanelekenns** [comment]
![image](https://user-images.githubusercontent.com/43152879/106977401-22be4f80-670f-11eb-9db7-d1783f44ea8f.png)

Hi there, this may be obvious to some, but I have not been able to figure out how to actually use the extension. When I try to select a repository in the bar as shown in the image, no options appear. I tried typing out the path from the workspace directory to the root of my repository and pressing enter, but nothing happens, no feedback whatsoever.

As you can see the side bar for the extension is empty and I'm not really sure how to get things started.

If it helps, I am developing with the Remote -SSH development extension.

I'm really looking forward to using this as I got used to using Azure's pull request diff comparison between branches, but formal PRs aren't part of my current team's workflow. Any help that can be provided is much appreciated.

**Comment by letmaik** [comment]
@keanelekenns I'm using the remote ssh extension as well, so that can't be it. The extension queries the git extension for all open repositories and then applies a filter to only show those that are within (or the same as) one of the workspace folders, which should nearly always be the case. Can you think of anything that's unusual in your environment?

**Comment by keanelekenns** [comment]
I discovered that it is not able to handle repositories inside a symbolic link, so I had to add the folder with the repositories to the workspace. I am now able to select repositories with the extension, but still nothing appears in the panel to actual view any diffs. I tried refreshing the tree and got this message:
![image](https://user-images.githubusercontent.com/43152879/107069582-036b0500-6797-11eb-9442-98b586f321f9.png)

What is the extension trying to do with this file?

**Comment by letmaik** [comment]
I never tested with symlinks, but I could imagine it causes issues. Can you create an issue for that?

Regarding the error message, could you post the log from the Output pane? That should show all the git commands that are run and may help with diagnosis.

**Comment by keanelekenns** [comment]
![image](https://user-images.githubusercontent.com/43152879/107081135-1554a400-67a7-11eb-85ce-7064c63cf1f1.png)
It continuously outputs the last line (I guess it's going through all the files trying to detect changes)

**Comment by letmaik** [comment]
Which git version do you have? Also, can you run `git rev-parse --git-common-dir` manually in the repo folder?

**Comment by keanelekenns** [comment]
Yeah that looks like the problem, sorry to be a bother, but thank you for helping. I imagine it will work if I update it.
![image](https://user-images.githubusercontent.com/43152879/107090001-661ec980-67b4-11eb-9bda-f01640d397ae.png)

**Comment by letmaik** [comment]
Yeah, that version is 9 years old :)

**Comment by dieterlohrstraeter** [comment]
Hi @letmaik, I had the same problem when I tried to use it for the first time. Would be better to leave a few words about how to start with comparing two branches. I needed to try out a lot until I found how it works.
